hazelcast/src/main/java/com/hazelcast/config/TSInMemoryTierConfig.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import com.hazelcast.internal.config.ConfigDataSerializerHook;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * In-Memory tier configuration of Tiered-Store.
+ *
+ * @since 5.1
+ */
+public class TSInMemoryTierConfig implements IdentifiedDataSerializable {
+
+    /**
+     * Default capacity in bytes.
+     */
+    public static final long DEFAULT_CAPACITY = 256L << 20;
+
+    private long capacity = DEFAULT_CAPACITY;
+
+    public TSInMemoryTierConfig() {
+
+    }
+
+    public TSInMemoryTierConfig(TSInMemoryTierConfig tsInMemoryTierConfig) {
+        capacity = tsInMemoryTierConfig.getCapacity();
+    }
+
+    /**
+     * Returns the capacity of this in-memory tier.
```

Comment:
unit of capacity?

Reply:
thanks. Now the capacity is a `MemorySize` rather than long.